docs/eog/case-studies-cw20-infrastructure.rst

```diff
@@ -0,0 +1,67 @@
+We also provided `guidance <https://software.ac.uk/news/collaborations-workshop-2020-cw20-move-online>`_ and documentation on reclaiming costs to help participants who had already booked their travel to Belfast.
+
+For the in-person event, we had planned to use additional tools such as `Slack <https://slack.com>`_, `Slido <https://www.sli.do/>`_ and `uCONFLY <http://uconfly.org/>`_, however for the online event we felt that it was important to minimise the number of platforms that participants had to register with and keep track of. 
+We narrowed our needs down to platforms for video conferencing, a chat system for sharing information and community engagement, and a tool for collaborative note-taking to keep everyone synchronised. 
+
+Platforms
+----------
+
+Video conferencing
+^^^^^^^^^^^^^^^^^^^
+
+`Zoom <https://zoom.us/>`_ was very quickly identified as the video conferencing platform that we would use because we had the most experience with it and it had the functionality we needed (namely breakout rooms and easy recording of the event). 
+We initially planned to use a single Zoom meeting room to contain the entire event (with breakout rooms used for the Discussion, Collaborative Ideas, and Mini-workshop sessions). 
+However, workshop facilitators also wanted to make use of the breakout room feature for their sessions, which wouldn’t be possible if they were *already in* breakout rooms. 
+We therefore obtained five Zoom Host accounts in total to accommodate this: one for the main/plenary room, plus four for the mini-workshops which would take place in parallel. 
+
+Chat system
+^^^^^^^^^^^^
+
+Although Zoom has chat functionality, it can be distracting for hosts/speakers and difficult to manage when there are many participants using it. 
+We decided to disable the Zoom chat (except for the case where participants could communicate directly with the meeting Host) and direct engagement and communication to the CW20 Slack workspace, as this would allow participants to connect even when the Zoom room is closed. 
+A communication channel that persists outside the meeting is helpful for maintaining access to resources shared, referencing discussions and facilitating community engagement. 
+We created practical channels for event organisation and communication, such as a ``#help-desk`` channel for participants to direct their technical issues to and private backchannels for the organisers, helpers, and Code of Conduct committee. 
+Participants were able to create channels for their Discussion groups and Hackday teams. 
+Slack also allowed for a more human and informal engagement through the creation of channels such as ``#pets-at-cw20``, ``#kids-at-cw20`` and ``#outside-my-window-cw20`` for sharing photos of our pets, children and working from home environment during lockdown.
+
+
+Collaborative documenting
```

Comment:
Is it worth mentioning the limit we reached with Google Docs - i.e. 100 connections before you had to make it public to support that many connections - and we could not do that as it would open the event to disruption. And perhaps a caveat that at the time of writing HackMD although requiring the knowledge of some syntax can support over 240+ connections.